package.json

```diff
@@ -64,7 +64,7 @@
     "tv4": "^1.2.7"
   },
   "optionalDependencies": {
-    "cdmiclient": "scality/cdmiclient#8f0c2e6"
+    "cdmiclient": "git+https://github.com/scality/cdmiclient#8f0c2e6"
```

Comment:
Made the requested updates on eve to make it work with the `scality/cdmiclient#8f0c2e6` syntax.

Could you please make the change?